apis/server_management/scripts/wazuh_server_management_apid.py

```diff
@@ -71,14 +71,6 @@ def spawn_process_pool():
     signal.signal(signal.SIGINT, signal.SIG_IGN)
 
 
-def spawn_events_pool():
```

Comment:
Why are we removing the events pool exactly? Are we going to handle the request like any other?

Reply:
In such a case, any non-specific defined function will be processed at the default `process_pool` process pool

Reply:
Right, I didn't remember we removed that endpoint :)